components/classroom/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="./styles.css" />
+    <title>"TPEN Classroom Interface: Home"</title>
+</head>
+    <body>
+        <div class="header">
+            <h1>TPEN Classroom Interface</h1>
+            <div class="hamburger-menu" onclick="toggleMenu()">
+                &#9776 <!-- Hamburger icon -->
+            </div>
+        </div>
+            <div class ="navbar" id="navbar">
+            <!--navigation bar below: we can try making this collapsible on mobile devices?
+            placeholder categories, will be changed as we solidify the pages needed
+            -->
```

Comment:
you can remove my comment! i didn't realize you were going to do the collapsible menu